controllers/workspace/validation.go

```diff
@@ -20,15 +20,12 @@ import (
 	"github.com/devfile/devworkspace-operator/pkg/webhook"
 )
 
-// validateCreatorTimestamp checks that a devworkspace was created after workspace-related mutating webhooks
+// validateCreatorLabel checks that a devworkspace was created after workspace-related mutating webhooks
 // and ensures a creator ID label is applied to the workspace. If webhooks are disabled, validation succeeds by
 // default.
 //
 // If error is not nil, a user-readable message is returned that can be propagated to the user to explain the issue.
-func (r *DevWorkspaceReconciler) validateCreatorTimestamp(workspace *devworkspace.DevWorkspace) (msg string, err error) {
-	if config.ControllerCfg.GetWebhooksEnabled() != "true" {
-		return "", nil
-	}
+func (r *DevWorkspaceReconciler) validateCreatorLabel(workspace *devworkspace.DevWorkspace) (msg string, err error) {
```

Comment:
Does it make sense to disable that check for non-restricted-access workspaces?

Reply:
Probably, but all workspaces get a creator label now; the check itself is doing something kind of different from before. The only way we would have a DW without the label would be if it was created before webhooks, in which case it may make sense to fail the workspace anyways since it might have missed conversion, etc.